src/components/ebay-button/index.marko

```diff
@@ -49,11 +49,21 @@ $ {
         !truncateClass && !fixedHeightClass && sizeClass,
         (priority === "secondary" ||
             priority === "primary" ||
-            priority === "delete") &&
+            priority === "delete" ||
+            priority === "tertiary") &&
             `${baseClass}--${priority}`
     ]
     data-ebayui=true
     type=(tag === "button" && (input.type || "button"))
-    aria-disabled=(input.partiallyDisabled && "true")>
-    <${input.renderBody}/>
+    aria-disabled=((input.partiallyDisabled || input.loading) && "true")
+    aria-live=(input.loading && "polite")
+    aria-label=(input.loading && "Loading...")>
+    <if(input.loading)>
```

Comment:
Im wondering, do we want to do an enum? Like `body-state="loading"`? or something like that.

Reply:
I'm not sure what an enum would look like here.. could you elaborate?